HOWTORELEASE.rst

```diff
@@ -0,0 +1,84 @@
+==================
+How to release tox
+==================
+
+This matches the current model that can be summarized as this:
+
+* tox has no long lived branches.
+
+* Pull requests get integrated into master by members of the project when they feel confident that this could be part of the next release. Small fix ups might be done right after merge instead of disscussing back and forth to get minor problems fixed. to keep the workflow simple.
+
+**Normal releases**: done from master when enough changes have accumaluted (whatever that means at any given point in time).
+
+**"Special" releases**: (in seldom cases when master has moved on and is not in a state where a quick release should be done from that state): the current release tag is checked out, the necessary fixes are cherry picked and a package with a patch release increase is built from that state. This is not very clean but seems good enough at the moment as it does not happen very often. If it does happen more often, this needs some rethinking (and rather into the direction of making less buggy releases than complicating release development/release process).
+
+HOWTO
+=====
+
+Prerequisites
+-------------
+
+* Push rights for https://github.com/tox-dev/tox
+* Release rights for https://pypi.org/project/tox/
+* [optional] An account on https://devpi.net to upload the package under test
+* A system with either tox + `bash <https://www.gnu.org/software/bash/>`_ or `vagrant <https://github.com/tox-dev/tox/blob/master/Vagrantfile>`_ (which contains tox + bash)
+* Accountability: if you cut a release that breaks the CI builds of projects using tox, you are expected to fix this within a reasonable time frame (hours/days - not weeks/months) - if you don't feel quite capable of doing this yet, partner up with a more experienced member of the team and make sure they got your back if things break.
+
+pypi/devpi configuration
+------------------------
+
+**note:** this is in a state of flux due to changes in pypi infrastructure and depending whether devpi push can be used or not. **Please keep this updated according to current process**
+
+**Current process** pypi upload packages from `dist/` via twine.
+
+If you want to use the scripts in `task/` you need a `.pypirc` with a correctly configured `pypi` section (see below). Otherwise just upload the release package which ever way you see fit.
+
+[pypi] section in `.pypirc` should look somehow like this::
```

Comment:
You can use:

```
.. code-block:: ini

    [pypi]
    ;repository=https://pypi.python.org/pypi
    ...
```

For syntax highlighting (you have to remove the `::` from the previous sentence).